### ISSUE TITLE
bump down Turing.jl

### ISSUE THREAD

**Issue by terasakisatoshi** [terasakisatoshi/MyWorkflow.jl#114]
I think it is a issue of Turing.jl v0.13.0

```
┌ Warning: Package Bijectors does not have LinearAlgebra in its dependencies:
│ - If you have Bijectors checked out for development and have
│   added LinearAlgebra as a dependency but haven't updated your primary
│   environment's manifest file, try `Pkg.resolve()`.
│ - Otherwise you may need to report an issue with Bijectors
└ Loading LinearAlgebra into Bijectors from project dependency, future warnings for Bijectors are suppressed.
┌ Warning: Error requiring DistributionsAD from Bijectors:
│ LoadError: ArgumentError: Package Bijectors does not have Distributions in its dependencies:
│ - If you have Bijectors checked out for development and have
│   added Distributions as a dependency but haven't updated your primary
│   environment's manifest file, try `Pkg.resolve()`.
│ - Otherwise you may need to report an issue with Bijectors
│ Stacktrace:
│  [1] require(::Module, ::Symbol) at ./loading.jl:905
│  [2] include(::Module, ::String) at ./Base.jl:377
│  [3] include(::String) at /root/.julia/packages/Bijectors/bHaf6/src/Bijectors.jl:1
│  [4] top-level scope at none:0
│  [5] eval at ./boot.jl:331 [inlined]
│  [6] eval at /root/.julia/packages/Bijectors/bHaf6/src/Bijectors.jl:1 [inlined]
│  [7] (::Bijectors.var"#115#136")() at /root/.julia/packages/Requires/qy6zC/src/require.jl:85
│  [8] err(::Any, ::Module, ::String) at /root/.julia/packages/Requires/qy6zC/src/require.jl:42
│  [9] (::Bijectors.var"#114#135")() at /root/.julia/packages/Requires/qy6zC/src/require.jl:84
│  [10] withpath(::Any, ::String) at /root/.julia/packages/Requires/qy6zC/src/require.jl:32
│  [11] (::Bijectors.var"#113#134")() at /root/.julia/packages/Requires/qy6zC/src/require.jl:83
│  [12] listenpkg(::Any, ::Base.PkgId) at /root/.julia/packages/Requires/qy6zC/src/require.jl:15
│  [13] macro expansion at /root/.julia/packages/Requires/qy6zC/src/require.jl:81 [inlined]
│  [14] __init__() at /root/.julia/packages/Bijectors/bHaf6/src/Bijectors.jl:266
│ in expression starting at /root/.julia/packages/Bijectors/bHaf6/src/compat/distributionsad.jl:5
└ @ Requires ~/.julia/packages/Requires/qy6zC/src/require.jl:44
```

**Comment by terasakisatoshi** [comment]
Plan: bump it down -> 0.12.0

**Comment by terasakisatoshi** [comment]
#113 will solve this issue

**Comment by terasakisatoshi** [comment]
closed via #113
